aten/src/THCS/generic/THCSTensor.cpp

```diff
@@ -120,6 +126,29 @@ THCSTensor* THCSTensor_(_set)(THCState *state, THCSTensor *self, THCIndexTensor
   return THCSTensor_(_move)(state, self, THCIndexTensor_(newClone)(state, indices), THCTensor_(newClone)(state, values));
 }
 
+THCSTensor* THCSTensor_(_move_csr)(THCState *state, THCSTensor *self, THCudaIntTensor *csr) {
+  int empty = (csr == NULL);
```

Comment:
Any reason to keep it as a variable and not as an `if` condition?

Reply:
The only reason is parallelism with https://github.com/pytorch/pytorch/pull/6225/files/358d3416df470f680bf7d361a0cb859ba77167af#diff-ab3b67ab37afe194592f04e8ad1b11dcR100. Should we modify both?